### PR TITLE
Kssmultisrc

### LINKED ISSUE #16: Fix multi source KSS situation.
```
{
  "source": "styleguide",
  "destination": "build/styleguide",
  "template": "node_modules/kss-bootstrap/bootstrap",
  "css": "build/styleguide.css"
}
```
Currently the source key points to "styleguide" as opposed to say "src/styles/"
Which takes the bootstrap kss nicely, but in fact leaves out all KSS documentation in the project itself.

is it possible that source be an array or sources?

@RobLoach, thoughts?

### PR DESCRIPTION
**Pull request by madeofpeople**
Fixes #16

### CODE REVIEW

**Comment by madeofpeople** [comment]
This involves not going the .node-kss.json route, but configuring the styeguide script in package.json as per https://github.com/kss-node/kss-node/issues/136

**Comment by madeofpeople** [comment]
@andrewmallis this pull request has nothing to do with menu links.
If you follow the conversation surrounding this ticket you will note that there is no way to provide multiple sources to kss via the .node-kss.json method but indeed via the command line method. 

So instead of replying on the configuring a json file and running node-kss --config path-to-json this fix now does node-kss --source aSource --source anotherSource --source anotherSource --css ... --template ... etc.... as per  https://github.com/kss-node/kss-node/issues/136

**Comment by madeofpeople** [comment]
Sticky footer kss was removed because it was getting in the way of testing.
We haven't really used a sticky footer anywhere.
We can re-introduce it in a feature branch later.

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "test": "./node_modules/.bin/metalsmith build && npm run-script styleguide",
     "postinstall": "./node_modules/.bin/bower install",
     "start": "./node_modules/.bin/grunt",
-    "styleguide": "./node_modules/kss-bootstrap/node_modules/.bin/kss-node --config='.kss-node.json'"
+    "styleguide": "./node_modules/kss-bootstrap/node_modules/.bin/kss-node --source styleguide --source src/styles --destination build/styleguide --css build/styles/main.css --template node_modules/kss-bootstrap/bootstrap"
```
Comment (RobLoach) [inline package.json]: So glad that KSS-Node takes multiple sources now :+1: .

Comment (madeofpeople) [inline package.json]: You sir, planted that seed.

**Comment by madeofpeople** [comment]
@RobLoach this is failing because the --css flag is deprecated... @JQTNguyen was having this problem.

**Pull request merged**